src/Kinepolis/KinepolisService.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Kinepolis;
+
+final class KinepolisService
```

Comment:
The reason for a service is to combine all the work that is done inside the CLI command and inject this with interfaces. So getting movies, parsing those movies, and dispatching commands should all happen here.

Because the client and parser are interfaces they can be mocked and the commands passed on the recording command bus can be validated.